lib/game.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "display"
+require_relative "messages"
+require_relative "rules"
+require_relative "player"
+
+class Game
+
+  def initialize(player_one, player_two, display)
+    @player_one = player_one
+    @player_two = player_two
+    @display = display
+  end
+
+  def run
+    player_prompt
+    player_choices
+    determine_winner
+    winner
+  end
+
+  def player_prompt
+    @display.present(Messages.new.user_prompt)
```

Comment:
Love the messages class! I wouldn't create a new instance each time; I'd instantiate one in the initialize method that I could call on each time.